pkg/infra/filestorage/cdk_blob_filestorage.go

```diff
@@ -22,15 +22,16 @@ type cdkBlobStorage struct {
 	rootFolder string
 }
 
-func NewCdkBlobStorage(log log.Logger, bucket *blob.Bucket, rootFolder string, pathFilters *PathFilters) FileStorage {
+func NewCdkBlobStorage(log log.Logger, bucket *blob.Bucket, rootFolder string, pathFilters *PathFilters, supportedOperations []Operation) FileStorage {
```

Comment:
Nit. Should name refer to GCP rather than Cdk (cloud development kit)? In addition would move this to a separate sub-package. No biggie, but something to consider for the future.